docker/tools/run_ccf_tests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests are run EXCLUSIVELY with all services running on localhost
+export PDO_HOSTNAME=localhost
+export PDO_LEDGER_URL=http://localhost:6600
+
+source /project/pdo/tools/environment.sh
+source ${PDO_HOME}/ccf/bin/lib/pdo_common.sh
+
+export no_proxy=$PDO_HOSTNAME,$no_proxy
+export NO_PROXY=$POD_HOSTNAME,$NO_PROXY
```

Comment:
because some people (not to name names) keep messing up their no_proxy configurations. the rest of the no_proxy config maybe messed up, but this should be enough to ensure that the tests run correctly.